lib/web/ui/integration.go

```diff
@@ -98,6 +98,8 @@ type ResourceTypeSummary struct {
 	ResourcesEnrollmentSuccess int `json:"resourcesEnrollmentSuccess,omitempty"`
 	// DiscoverLastSync contains the time when this integration tried to auto-enroll resources.
 	DiscoverLastSync *time.Time `json:"discoverLastSync,omitempty"`
+	// UnresolvedUserTasks contains the count of unresolved user tasks related to this integration and resource type.
+	UnresolvedUserTasks int `json:"unresolvedUserTasks,omitempty"`
```

Comment:
Do we want `omitempty` here? Wouldn't it be easier for the UI if the field was always present, even if its value is 0?